theme/less/components/tabs.less

```diff
@@ -63,11 +63,10 @@ You can use tabs from DSFR. It uses DSFR colors and conventions. It allows you t
 
 .tabs {
     display: flex;
+    flex-wrap: wrap;
 
     .tab {
-        + .tab {
-            margin-left: @spacing-sm;
-        }
+        margin: @spacing-xxs;
```

Comment:
We can remove this line here and use DSFR utility class on reuse/list and home to allow for different margins (`.fr-m-1v` on reuse/list and `.fr-ml-3v` on home to keep the previous layout).

Reply:
We'll use DSFR tags instead following https://github.com/etalab/udata-front/pull/57.